src/cli/build.ts

```diff
@@ -139,6 +139,8 @@ export default function (program: RootCmd) {
       await fs.writeFile(yamlPath, yaml);
       await fs.writeFile(zarfPath, zarf);
 
+      await assets.generateHelmChart(outputDir);
```

Comment:
Hmm that is a good point. There was not a reason for it other than I thought it would be a lot cleaner on the Asset class since it is using several class members there.

Reply:
could be two functions for sure though

Reply:
Yeah, I'm not too particular about which way is / would be better -- it just stood out to me as "different" & so I wondered why.